README.md

```diff
@@ -41,9 +41,7 @@ header:
 
 **NOTE**: The full configurations can be found in [the configuration section](#configurations).
 
-### Use as a Binary
-
-#### Install
+### Compile from Source
```

Comment:
We will have another 2 sections: "Use with Docker image", "Use as a Binary" after we have the release binary and docker image